composer/datasets/in_context_learning_evaluation.py

```diff
@@ -193,11 +201,15 @@ def prep_examples(self, num_fewshot: int, prompt_string: str, example_delimiter:
             if num_fewshot > 0:
                 fewshot_idxs = _get_fewshot_sample_idxs(len(self.samples), num_fewshot, sample_idx, fewshot_rng)
                 for fewshot_idx in fewshot_idxs:
-                    ctxt, cont = self.samples[fewshot_idx]['context'], self.samples[fewshot_idx]['answer']
+                    ctxt, cot, cont = (self.samples[fewshot_idx]['context'],
```

Comment:
Separate this into 3 lines and use more descriptive variable names, e.g. `context` `cot` and `answer`

Reply:
And similarly for variables below.